daemon/algod/api/client/encoding.go

```diff
@@ -60,5 +60,5 @@ func (a *ChecksumAddress) UnmarshalText(text []byte) error {
 // MarshalText implements the encoding.TextMarshaler interface
 func (a ChecksumAddress) MarshalText() (text []byte, err error) {
```

Comment:
Any reason we need to keep this package's `ChecksumAddress` type or can we get rid of it too?

Reply:
AFAIK, no. Will remove ASAP